lib/docker2aci.go

```diff
@@ -447,6 +442,20 @@ func generateManifest(layerData DockerImageData, dockerURL *ParsedDockerURL) (*s
 	return genManifest, nil
 }
 
+func getExecCommand(entrypoint []string, cmd []string) types.Exec {
+	var command []string
+	if entrypoint == nil && cmd == nil {
+		return nil
+	}
+	command = append(entrypoint, cmd...)
+	// non-absolute paths are not allowed, fallback to "/bin/sh -c command"
```

Comment:
Starting to feel like we should have a means to provide warnings during the conversion process when things like this happen, thoughts?  (In the real world this would _probably_ very rarely cause problems since having /bin/sh in Docker images is a really common pattern, but would be good practise to give people _some_ hint that we are doing this kind of translation and the produced image is potentially non-functional)